apps/hobbies-helsinki/src/pages/_app.tsx

```diff
@@ -96,22 +89,20 @@ function MyApp({ Component, pageProps }: AppProps<CustomPageProps>) {
       <RHHCConfigProvider config={rhhcConfig}>
         <TopProgressBar />
         <ApolloProvider client={cmsApolloClient}>
-          <ApolloProvider client={eventsApolloClient}>
```

Comment:
I need to test this removal once more. At least it does not make sense to clear the context set by cmsClient!!

Reply:
I think it's better like this than to add a client to every query-hook. Less code and the component is more movable.